### PR TITLE
Job submission on slurm

### LINKED ISSUE #190: Add routes and logic for job submission with SLURM on plgrid
In the first iteration we should assume that:
1. `shieldhit` binary is installed in $SCRATCH (but this should configurable, now lets hardcode that)
2. `convertmc` binary is installed in $SCRATCH (but this should configurable, now lets hardcode that)
3. user has a computing grant and we use a default grant settings
4. walltime is hardcoded to 59 minutes
5. we use single node and single task (`-n 1` in slurm)

We also need a bash script for submission which:
1. loads the GCC with which shieldhit was compiled (module load...)
2. adds $SCRATCH to $PATH, so shieldhit and convertmc binaries are visible
3. creates a temporary directory in $SCRACH with input files (plgdata needed ??)
4. changes working directory to 3.
5. calls shieldhit with the input files

Once the simulation is done output files (*.bdo) should be transmitted back to the backend (plgdata ???) and digested by pymchelper and converted to JSON.

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,14 @@ curl -i -X GET -b cookies.txt -H "Content-Type:application/json" -d "{'task_id'
 
 Although it might be inefficient way of testing so there is a prepared example ``call_api_example.py`` in yaptide/examples folder
 
+# PLGrid
+
+TODO: write more
```
Comment (grzanka) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -130,6 +130,14 @@ curl -i -X GET -b cookies.txt -H "Content-Type:application/json" -d "{'task_id'
...
+Command generating `grid_proxy` file on Linux (WSL also works fine)
```
Comment (grzanka) [inline README.md:135]: Add a comment mentioning that username needs to be adjusted

File: plgrid/rimrock_bash.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+shieldhit_bash: str = """#!/bin/bash
+#SBATCH --nodes 1
+#SBATCH --ntasks 1
+#SBATCH --time=00:04:59
+#SBATCH -A plgccbmc11-cpu
+
+JOB_ID=$(printenv | grep -Po '(?<=^SLURM_JOBID=)\\w*$')
+SUBMIT_DIR=$(pwd)
+SCRATCH_DIRECTORY=/net/pr2/scratch/people/${{USER}}/${{JOB_ID}}
```
Comment (grzanka) [inline plgrid/rimrock_bash.py]: Rename `SCRATCH_DIRECTORY` to `WORKSPACE`, now the name is confusing as we already have a $SCRATCH directory

Comment (grzanka) [inline plgrid/rimrock_bash.py]:
```suggestion
SCRATCH_DIRECTORY=${{SCRATCH}}/${{JOB_ID}}
```

If I login to ares I already have a location to SCRATCH directory stored under $SCRATCH

File: yaptide/examples/input_files/beam.dat [hunk 4]
```diff
@@ -0,0 +1,9 @@
+RNDSEED      	89736501     ! Random seed
```
Comment (grzanka) [inline yaptide/examples/input_files/beam.dat:1]: replace snowman with pencil beam example !

File: plgrid/rimrock_bash.py (hunk 3 above)
Comment (p1003) [reply plgrid/rimrock_bash.py]: I have to check it, because for now it might not be available via rimrock

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**